tests/Browser/ClientTest.php

```
@@ -39,18 +39,20 @@ public function it_user_can_see_clients_on_client_index_and_go_to_the_customer_w
                 ->assertSee($client->primary_contact->name)
                 ->assertSee($client->company_name);
         });
+
+        /* Assert */
     }
```

PHPMD (2.15.0)

[error] The method it_shows_clients_on_index_page_with_navigation_links is not named in camelCase. (undefined)

(CamelCaseMethodName)